packages/vite-plugin-checker/src/checkers/eslint/main.ts

```diff
@@ -81,8 +81,9 @@ const createDiagnostic: CreateDiagnostic<'eslint'> = (pluginConfig) => {
 
       const handleFileChange = async (filePath: string, type: 'change' | 'unlink') => {
         const absPath = path.resolve(root, filePath)
+        const isChangedFileIgnored = await eslint.isPathIgnored(filePath)
 
-        if (type === 'unlink') {
+        if (type === 'unlink' || isChangedFileIgnored) {
           manager.updateByFileId(absPath, [])
         } else if (type === 'change') {
```

Comment:
Should `isChangedFileIgnored` also added to `change` type?

Reply:
Originally I did it this way in case the ESLint config changed (from ESLint's perspective, deleting a file and adding a file to the ignore list are very similar). I just tested this out and adding a file to the ignore list when the project is already running doesn't seem to work anyway. So, I just pushed a change to use an early return like you showed.